.github/workflows/autobuild.yml

```diff
@@ -3,123 +3,132 @@
 on:
   push:
     tags:
-      - "r*" # run this action if a tag beginning with r is created
-  workflow_dispatch:
-
+      - "r*"
+      - latest
+    branches:
+      - "*"
+#      - master         # could be restricted to master, if all branches take too much build time
 name:                               Publish Release
-jobs:
-
-  ## Creates the releases on GitHub ##
+jobs: 
```

Comment:
You've changed a lot and I still have to look through it. But it seems to work. Would it be possible that you add the same comments as I did?

Reply:
done bf47be2d516a9f592fe8bf3fa5615a3dca5818ab

Reply:
at lease I did my best, feel free to adjust them to your needs